### PR TITLE
Don't call onMissedHeartbeat if connection is closed #112



### LINKED ISSUE #112: Don't call onMissedHeartbeat if connection is closed

### PR DESCRIPTION
**Pull request by jkolobok**

### CODE REVIEW

File: src/main/java/net/openhft/chronicle/network/TcpEventHandler.java [hunk 1]
```diff
@@ -59,6 +59,7 @@
         implements EventHandler, TcpEventHandlerManager<T> {
 
     public static final int TARGET_WRITE_SIZE = Integer.getInteger("TcpEventHandler.targetWriteSize", 1024);
+    private static final boolean CALL_MISSED_HEARTBEAT_ON_DISCONNECT = Boolean.getBoolean("chronicle.network.callOnMissedHeartbeatOnDisconnect");
```
Comment (JerryShea) [inline src/main/java/net/openhft/chronicle/network/TcpEventHandler.java]: The standard is to use Jvm.getBoolean (the main difference is that Jvm.getBoolean does not require =true)

**Review by JerryShea — APPROVED** [review]
Makes sense!

File: src/main/java/net/openhft/chronicle/network/TcpEventHandler.java [hunk 2]
```diff
@@ -59,6 +59,7 @@
         implements EventHandler, TcpEventHandlerManager<T> {
 
     public static final int TARGET_WRITE_SIZE = Integer.getInteger("TcpEventHandler.targetWriteSize", 1024);
+    private static final boolean CALL_MISSED_HEARTBEAT_ON_DISCONNECT = Jvm.getBoolean("chronicle.network.callOnMissedHeartbeatOnDisconnect");
```
Comment (nicktindall) [inline src/main/java/net/openhft/chronicle/network/TcpEventHandler.java:62]: `callOnMissedHeartbeatOnDisconnect` should this be `callMissedHeartbeatOnDisconnect`

**Review by rogersimmons — APPROVED** [review]

File: src/main/java/net/openhft/chronicle/network/TcpEventHandler.java (hunk 2 above)
Comment (nicktindall) [reply src/main/java/net/openhft/chronicle/network/TcpEventHandler.java:62]: Actually no I see now

**Review by nicktindall — APPROVED** [review]

**Pull request merged**